web_src/css/markup/codepreview.css

```
@@ -0,0 +1,26 @@
+.markup .code-preview-container {
+  border: 1px solid var(--color-secondary);
+}
+
+.markup .code-preview-container .code-preview-header {
+  border-bottom: 1px solid var(--color-secondary);
+  padding: 0.5em;
+  font-size: 11px;
+}
+
+.markup .code-preview-container table {
+  width: 100%;
+  margin: 0.25em 0;
+  max-height: 100px;
+  overflow-y: scroll;
```

```suggestion
  overflow-y: auto;
```

`scroll` always shows a scrollbar, even when not needed.